appveyor.yml

```diff
@@ -0,0 +1,19 @@
+build: false
+
+environment:
+    PYTHON: "C:\\Python27"
+    PYTHON_VERSION: "2.7.15"
+    PYTHON_ARCH: "32"
+
+    matrix:
+        - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
```

Comment:
Even the latest MSVC isn't sufficiently standards compliant to build the Adjacency_list test.

`error C2651: 'defer<meta::v1::detail::_and_<false>::invoke,defer<meta::v1::detail::_and_<false>::invoke,std::is_same<std::integer_sequence<bool,ranges::v3::concepts::models<ranges::v3::concepts::Readable,T>::type::value,ranges::v3::concepts::models<ranges::v3::concepts::Readable,_if_<meta::v1::list<std::integral_constant<bool,B>,ranges::v3::detail::projected_<I1,P1>>,std::true_type>::type>::type::value>,std::integer_sequence<bool,ranges::v3::concepts::models<ranges::v3::concepts::Readable,T>::type::value||,ranges::v3::concepts::models<ranges::v3::concepts::Readable,_if_<meta::v1::list<std::integral_constant<bool,B>,ranges::v3::detail::projected_<I1,P1>>,std::true_type>::type>::type::value||>>,meta::v1::defer<meta::v1::invoke,F,_if_<meta::v1::list<std::integral_constant<bool,B>,ranges::v3::detail::projected_<I0,P0>>,std::true_type>::type,_if_<meta::v1::list<std::integral_constant<bool,B>,ranges::v3::detail::projected_<I1,P1>>,std::true_type>::type>>::type,ranges::v3::concepts::models<ranges::v3::concepts::CopyConstructible,C>>::type': left of '::' must be a class, struct or union`

Whereas clang works fine.

Reply:
I have removed AppVeyor for now and it is probably best if it is just re-enabled when `MSVC` can build it 👍